milestone_project_1_original.py

```diff
@@ -0,0 +1,114 @@
+"""
+A simple Tic-Tac-Toe game.
+"""
+
+BOARD = [' ', ' ', ' ',
+         ' ', ' ', ' ',
+         ' ', ' ', ' ']
+
+CURRENT_PLAYER = "X"
+
+GAME_ON = True
+
+WINNER = " "

```

Comment:
unnecessary since you have CURRENT_PLAYER

Reply:
Fixed in 2d07b9c6673936279db3eb63b42d616242c0231e